docs/quickstart.md

```diff
@@ -169,85 +162,26 @@ Confirm password:
 
 v1.2.0
 
-AEA configurations successfully initialized: {'author': 'fetchai'}
+AEA configurations successfully initialized: {'author': 'valory'}
 ```
 
 <div class="admonition note">
   <p class="admonition-title">Note</p>
   <p>If you would rather not create an account on the registry at this point, then run <code>aea init --local</code> instead.</p>
 </div>
 
+
+
 ## Echo skill demo
 
 This is a simple demo that introduces you to the main components of an AEA. 
 
 The fastest way to have your first AEA is to fetch one that already exists!
 
 ``` bash
-aea fetch fetchai/my_first_aea:0.27.0
```

Comment:
Note, we no longer have the `aea interact` command in the `open-aea`. So the user has to use `echo` to file instead